src/styles/Cart.module.css

```diff
@@ -0,0 +1,137 @@
+.cart {
+  width: calc(100% - 324px);
+  padding: 24px 32px;
+  display: flex;
+  flex-direction: column;
+}
+
+.empty {
+  margin-top: 24px;
+  font-size: 24px;
+  line-height: 30px;
+  color: var(--light);
```

Comment:
wtf is this ? i know right answer - this code was copy pasted. please use one type of variables

Reply:
This crap is left over from the wrong merge, and enough with the copied code, there's a limit to everything